addons/mail/static/tests/systray/systray_activity_menu_tests.js

```diff
@@ -193,10 +193,18 @@ QUnit.test('activity menu widget: activity view icon', async function (assert) {
 
     // click on the "Issue" activity icon
     await testUtils.dom.click(activityMenu.$('.dropdown-toggle'));
+    assert.hasClass(activityMenu.$('.dropdown-menu'), 'show',
+        "dropdown should be expaned");
+
     await testUtils.dom.click(activityMenu.$(".o_mail_activity_action[data-model_name='Issue']"));
+    assert.doesNotHaveClass(activityMenu.$('.dropdown-menu'), 'show',
+        "dropdown should be collapsed");
 
     // click on the "Note" activity icon
+    await testUtils.dom.click(activityMenu.$('.dropdown-toggle'));
     await testUtils.dom.click(activityMenu.$(".o_mail_activity_action[data-model_name='Note']"));
+    assert.doesNotHaveClass(activityMenu.$('.dropdown-menu'), 'show',
```

Comment:
remove this assertion, already tested before

Reply:
Removed the assertion.